.gitlab/container_build/docker_linux.yml

```diff
@@ -23,6 +23,7 @@
     - crane flatten -t ${TARGET_TAG} ${TARGET_TAG}
   # Workaround for temporary network failures
   retry: 2
+  timeout: 45m
```

Comment:
This might even be a bit to preservative, AFAICS most non Windows jobs complete in about 5 to 15 minutes

Reply:
Indeed the p99 is 13m for successful jobs, I'll change it to 30m as the runner may take 10m to be found